packages/whale-api-client/src/api/poolpairs.ts

```diff
@@ -265,6 +270,7 @@ export interface SwapPathPoolPair {
     ab: string
     ba: string
   }
+  estimatedDexFees?: EstimatedDexFees
```

Comment:
```suggestion
  dexFees?: DexFees
```

Looking at the implementation now, there's no actual "estimation" going on
Hence I think we can simplify the name